localization/english/CWP/africa/CWP_libya_l_english.yml

```
@@ -0,0 +1,140 @@
+  libya.4.f: "There is no state with a democracy except Libya on the whole planet.\n― Muammar Gaddafi"
+  libya.4.a: "And now?"
+  libya.4.tt: "If Libya tries to annex us, we can declare our independence by force"
+
+  libya.5.t: "Libyan Ambitions"
+  libya.5.d: "Muammar Gaddafi, the leader of Libya, sets his sights on expanding his influence far beyond his nation's borders. With unwavering determination, Gaddafi seeks to bring our country into his fold, forming an unprecedented union that could potentially reshape the geopolitical landscape."
+  libya.5.f: "When I met Nasser, he said to me, 'I see myself when I was young in you. You are the future for the Arab revolution.' This meant very, very much to me.\n― Muammar Gaddafi"
+  libya.5.a: "Are we in danger?"
+  libya.5.tt: "A new journal entry becomes available"
+
+  libya.6.t: "[SCOPE.sCountry('gaddafi_unification_target_scope').GetName] joins our Union"
+  libya.6.d: "In the face of relentless diplomatic pressure, [SCOPE.sCountry('gaddafi_unification_target_scope').GetName] has taken a momentous step towards embracing unity, deciding to join our distinguished Union. Despite their initial resilience, the unyielding force of diplomatic persuasion has paved the way for their integration."
+  libya.6.f: "There is no state with a democracy except Libya on the whole planet.\n― Muammar Gaddafi"
+  libya.6.a: "The union grows"
+
+  libya.7.t: "[SCOPE.sCountry('gaddafi_unification_target_scope').GetName] rejects our Offer"
+  libya.7.d: "In a resolute stand for its sovereignty and independence, [SCOPE.sCountry('gaddafi_unification_target_scope').GetName] firmly declines Colonel Gaddafi's proposition to join his envisioned union. Despite diplomatic pressures and enticing promises, the nation remains steadfast in its refusal, seeing the proposal as a threat to its hard-won autonomy."
+  libya.7.f: "The times of Arab nationalism and unity are gone forever. These ideas which mobilized the masses are only a worthless currency. Libya has had to put up with too much from the Arabs for whom it has poured forth both blood and money.\n― Muammar Gaddafi"
+  libya.7.a: "Traitors!"
+
+  libya.8.t: "[SCOPE.sCountry('gaddafi_unification_target_scope').GetName] rejects our Offer"
+  libya.8.d: "In a tense turn of events, Muammar Gaddafi contemplates a daring coup attempt against the government of [SCOPE.sCountry('gaddafi_unification_target_scope').GetName], following their resounding refusal to join his proposed union. The rejection has sent shockwaves through diplomatic circles, exacerbating already strained relations between the neighboring nations."
+  libya.8.f: "I staged a coup d’état and the workers staged the revolution, occupying the factories, becoming partners instead of employees, eliminating the monarchic administration and setting up people’s committees. In short, they liberated themselves. Students did the same. Today in Libya, only the people count.\n― Muammar Gaddafi"
```

- removed Unicode style single-quotation marks as vic3 doesn't like this
- italic around quotes
```suggestion
  libya.8.f: "#italic I staged a coup d'état and the workers staged the revolution, occupying the factories, becoming partners instead of employees, eliminating the monarchic administration and setting up people's committees. In short, they liberated themselves. Students did the same. Today in Libya, only the people count.#!\n― Muammar Gaddafi"
```